### ISSUE TITLE
output_buildfile_suffix = "BUILD.bazel" does not work

### ISSUE THREAD

**Issue by prestonvanloon** [google/cargo-raze#101]
```
INFO: Call stack for the definition of repository 'raze__bls_aggregates__0_6_1':
 - /home/preston90/bls-benchmarks/cargo/crates.bzl:15:9
 - /home/preston90/bls-benchmarks/cargo/crates.bzl:72:5
 - /home/preston90/bls-benchmarks/WORKSPACE:49:1
ERROR: /home/preston90/bls-benchmarks/cargo/BUILD.bazel:11:1: no such package '@raze__bls_aggregates__0_6_1//': Traceback (most recent call last):
        File "/home/preston90/.cache/bazel/_bazel_preston90/5ab6d69733459142a1e1f37eacad08ea/external/bazel_tools/tools/build_defs/repo/git.bzl", line 228
                workspace_and_buildfile(ctx)
        File "/home/preston90/.cache/bazel/_bazel_preston90/5ab6d69733459142a1e1f37eacad08ea/external/bazel_tools/tools/build_defs/repo/utils.bzl", line 64, in workspace_and_buildfile
                ctx.symlink(ctx.attr.build_file, "BUILD.bazel")
Not a regular file: /home/preston90/bls-benchmarks/cargo/remote/bls-aggregates-0.6.1.BUILD and referenced by '//cargo:bls_aggregates'
```

Link 49 of the workspace is `raze_fetch_remote_crates()`. Something is still referencing the default BUILD file path.

**Comment by acmcarther** [comment]
Oops, thanks for reporting this and the extra details. #102 will fix this. I'll cut a release once I merge that (and will ping back here).

**Comment by acmcarther** [comment]
Fix is in latest published version 0.2.7.

**Comment by prestonvanloon** [comment]
Wow! Thanks for fast fix!
